framework/framework.go

```diff
@@ -128,25 +128,21 @@ func (f *Framework) DeleteFunc(obj interface{}) {
 
 	resourceSet, err := f.resourceRouter.ResourceSet(obj)
 	if err != nil {
-		f.logger.Log("error", fmt.Sprintf("%#v", err), "event", "delete")
+		f.logger.Log("event", "delete", "function", "DeleteFunc", "level", "error", "message", "stop framework reconciliation due to error", "stack", fmt.Sprintf("%#v", err))
 		return
 	}
 
 	ctx, err := resourceSet.InitCtx(context.Background(), obj)
 	if err != nil {
-		f.logger.Log("error", fmt.Sprintf("%#v", err), "event", "delete")
+		f.logger.Log("event", "delete", "function", "DeleteFunc", "level", "error", "message", "stop framework reconciliation due to error", "stack", fmt.Sprintf("%#v", err))
 		return
 	}
 
-	f.logger.LogCtx(ctx, "action", "start", "component", "operatorkit", "function", "ProcessDelete")
```

Comment:
I removed this because I came to realize this was never useful to me personally. Let me know when you think differently.

Reply:
I find those start/end messages useful. I wouldn't mind if they were a bit more standard (have level and such).